Capnp build error on Ubuntu: no such member Map

After the recent capnp removal, I install all dependencies, build capnp, and get the failure when building nupic.core: 

```
$ make -j4
[  0%] [  1%] [  1%] Compiling Cap'n Proto schema nupic/proto/SpatialPoolerProto.capnp
Compiling Cap'n Proto schema nupic/proto/Map.capnp
[  2%] Built target gtest
/home/mmm/devel/nupic/nupic.core/src/nupic/proto/Map.capnp:3:23: error: Parse error.
Compiling Cap'n Proto schema nupic/proto/NetworkProto.capnp
make[2]: *** [nupic/proto/Map.capnp.c++] Error 1
make[2]: *** Waiting for unfinished jobs....
/home/mmm/devel/nupic/nupic.core/src/nupic/proto/Map.capnp:3:23: error: Parse error.
/home/mmm/devel/nupic/nupic.core/src/nupic/proto/NetworkProto.capnp:4:39-42: error: No such member: Map
/home/mmm/devel/nupic/nupic.core/src/nupic/proto/NetworkProto.capnp:8:15-18: error: Not defined: Map
make[2]: *** [nupic/proto/NetworkProto.capnp.c++] Error 1
[  3%] Compiling Cap'n Proto schema nupic/proto/TemporalMemoryProto.capnp
make[1]: *** [CMakeFiles/nupic_core_solo.dir/all] Error 2
make: *** [all] Error 2
(env)mmm@mmm-U2442:~/devel/nupic/nupic.core/build/scripts$ git status 
On branch master
Your branch is up-to-date with 'origin/master'.
```

I think more detailed install info should be provided, I am building according to readme/Travis scripts...only difference is that I didnt set the LD_/CFLAGS - not sure how?

CC @scottpurdy

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -309,6 +309,9 @@ include_directories(${PYTHON_INCLUDE_DIRS})
 # Find and add capnp and kj and generate Cap'n Proto C++ files
 #
 set(CAPNPC_OUTPUT_DIR ${CMAKE_BINARY_DIR})
+set(CAPNPC_IMPORT_DIRS
+    ${CAPNPC_IMPORT_DIRS}
+    ${REPOSITORY_DIR}/src)
 find_package(CapnProto REQUIRED)
 include_directories(${CAPNP_INCLUDE_DIRS})
 add_definitions(${CAPNP_DEFINITIONS})

